source/monitoring.txt

```diff
@@ -12,8 +12,135 @@ Monitoring and Logging
+- `Using JConsole <https://docs.oracle.com/en/java/javase/17/management/using-jconsole.html>`__
+
+To learn more about tasks in {+kc+}, see the following resources:
+
+- `Kafka Connect Concepts from Confluent <https://docs.confluent.io/platform/current/connect/concepts.html#tasks>`__
+- `{+kc+} API documentation for Task interface <https://kafka.apache.org/21/javadoc/index.html?org/apache/kafka/connect/connector/Task.html>`__
+
+Enable Monitoring
+~~~~~~~~~~~~~~~~~
+
+The {+mkc+} uses {+kc+}'s metrics infrastructure to
+serve metrics. To read the metrics produced by your connector,
+enable JMX in your {+kc+} deployment.
+
+To learn how to enable JMX for a {+kc+} instance
+running on your host machine, see the
+`Official Kafka Documentation <https://kafka.apache.org/documentation/#monitoring>`__.
+
+To learn how to enable JMX for a containerized {+kc+}
+deployment, see
+`Kafka Monitoring and Metrics Using JMX with Docker <https://docs.confluent.io/platform/current/installation/docker/operations/monitoring.html>`__.
+
+.. _kafka-monitoring-example:
+
+Example - Monitor the Quick Start
+---------------------------------
+
+The sample environment provided in the Quick Start exposes metrics
+on your host machine at the URI ``localhost:35000``.
+
+To view these metrics with ``jconsole``, perform the following actions:
+
+.. tip:: Download jconsole
+
+   ``jconsole`` is part of the {+java-se+}. To download ``jconsole``, download the
+   `Java SE Development Kit <https://www.oracle.com/java/technologies/downloads/>`__
+   from Oracle.
+
+.. procedure::
+   :style: normal
+
+   .. step:: Start The Quick Start Pipeline and Add Connectors
+
+      Follow the Quick Start until the :ref:`<kafka-quick-start-send-a-document>`
+      step.
+
+   .. step:: Start jconsole
+
+      Run the following command from your command line to start ``jconsole``:
+
+      .. code-block:: shell
+
+         jconsole
+
+   .. step:: Connect to {+kc+} JMX Server
+
+      .. procedure::
+         :style: connected
+
+         .. step:: Enter Your JMX Server URI
+
+            Enter the URI ``localhost:35000`` into the :guilabel:`Remote Process`
+            text input box in the JConsole interface.
+
+         .. step:: Press the :guilabel:`Connect` button.
+
+         .. step:: Press the :guilabel:`Insecure Connection` button in the dialog box.
+
+   .. step:: Explore Metrics
+
+      .. procedure::
+         :style: connected
+
+         .. step:: Navigate to the :guilabel:`MBeans` tab in ``jconsole``.
+
+         .. step:: Inspect connector metrics.
+
+            Notice that the ``{+metrics-path+}.sink-task-0.records``
+            attribute has a value of ``0``.
+
+         .. step:: Continue the Quick Start
+
+            Continue the Quick Start until, but not through, the
+            :ref:`<kafka-quickstart-remove-the-sandbox>` step.
+
+            After completeing the preceding sections of the Quick Start,
```

Comment:
Question: Do we want readers to return to the console to see that the attribute has changed? If so, we should add that as a step.